tests/integration/throttle.go

```diff
@@ -312,13 +320,21 @@ func (s *CCVTestSuite) TestPacketSpam() {
 			infractionType = stakingtypes.Infraction_INFRACTION_DOUBLE_SIGN
 		}
 		valToJail := s.providerChain.Vals.Validators[ibcSeqNum%3]
-		packets = append(packets, s.constructSlashPacketFromConsumer(firstBundle, *valToJail, infractionType, ibcSeqNum))
+		packetsData = append(packetsData, s.constructSlashPacketFromConsumer(firstBundle, *valToJail, infractionType))
 	}
 
 	// Recv 500 packets from consumer to provider in same block
-	for _, packet := range packets {
-		consumerPacketData, err := provider.UnmarshalConsumerPacket(packet) // Same func used by provider's OnRecvPacket
+	var (
```

Comment:
I see these lines repeated a lot

```golang
timeoutHeight    = clienttypes.Height{}
timeoutTimestamp = uint64(firstBundle.GetCtx().BlockTime().Add(ccvtypes.DefaultCCVTimeoutPeriod).UnixNano())

packet := channeltypes.NewPacket(data, uint64(sequence),
			firstBundle.Path.EndpointA.ChannelConfig.PortID, firstBundle.Path.EndpointA.ChannelID,
			firstBundle.Path.EndpointB.ChannelConfig.PortID, firstBundle.Path.EndpointB.ChannelID,
			timeoutHeight, timeoutTimestamp)
```

Could we make a helper to make this all a one-liner? `NewPacketFromConsumer` or similar